utilities/conv_gefs2datm/comp_f77_code.csh

```diff
@@ -0,0 +1,8 @@
+#! /bin/csh
+module load intel/19.0.4.243
+module load impi/2019.0.4
+
+set name = $1
+#--- in hera 
+ifort -132 ${name}.f -o ${name} -L/apps/netcdf/4.6.1/intel/16.1.150/lib -lnetcdff
```

Comment:
1. There is no `include` in this line, it's impossible to compile this code!!!

2. updated compilation 
ifort $FFLAGS ${name}.f -o ${name} -I$NETCDF/include -L$NETCDF/lib -lnetcdff